src/system/game-data.ts

```diff
@@ -984,10 +983,9 @@ export class GameData {
       };
 
       if (!bypassLogin && !localStorage.getItem(`sessionData${slotId ? slotId : ""}_${loggedInUser?.username}`)) {
-        Utils.apiFetch(`savedata/session/get?slot=${slotId}&clientSessionId=${clientSessionId}`, true)
-          .then(response => response.text())
+        pokerogueApi.savedata.session.get({ slot: slotId, clientSessionId })
           .then(async response => {
-            if (!response.length || response[0] !== "{") {
+            if (!response && response?.length === 0 || response?.[0] !== "{") {
```

Comment:
```suggestion
            if (!response || response?.length === 0 || response?.[0] !== "{") {
```
Is `||` intended here instead of `&&`?